testdata/resolvers/tasks/http-resolver-task.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: http-resolver-task
+  annotations: 
+    tekton.dev/resolver: "http" 
+spec: 
+  steps: 
+    - name: http-request 
+      image: curlimages/curl:latest 
+      script: | 
+        #!/bin/sh 
+        echo "Making HTTP Request" 
+        curl -X GET https://github.com/openshift-pipelines/release-tests
```

Comment:
verify the curl response code.

Reply:
this test is not even supposed to use curl